.github/workflows/test.yaml

```diff
@@ -0,0 +1,34 @@
+# test.yml
+name: Testing
+
+on:
+  push:
+    branches:
+    - main
+  pull_request:
+    branches:
+    - main
+
+jobs:
+  python-linting:
+    runs-on: ubuntu-latest
+
+    steps:
+      - uses: actions/checkout@v3
+      - name: Set up Python 3.10
+        uses: actions/setup-python@v3
+        with:
+          python-version: "3.10"
```

Comment:
I don't know if there is any better way to pin a version to what we'll use in docker compose?

Reply:
Done!